electron/build-electron-release.sh

```diff
@@ -60,7 +60,7 @@ if [ -e "mac" ]; then
         mv "${PDT_NAME}-${APP_VERSION}.dmg" "../${PKG_NAME}-${APP_VERSION}-gui-electron-osx-x64.dmg"
     elif [ -e "${PDT_NAME}.app" ]; then
         if [[ "$OSTYPE" == "darwin"* ]]; then
-            tar czf "../${PKG_NAME}-${APP_VERSION}-gui-electron-osx-x64.zip" "${PDT_NAME}.app"
+            rm -rf "${PDT_NAME}.app"
```

Comment:
I think this line needs to be in the higher if block, so that it applies to all host OS build machines

Reply:
Actually we should remove the `npm run pack-mac` on linux, cause we now only need `dmg` for osx version. The `npm run pack-mac` is used on linux to build a osx `.app` file, and compress it as a `.zip` file. After removeing the osx app build on linux, we can remove the `OSTYPE` check here.

Reply:
Does linux/windows make a dmg file?